user-guide/grpc.md

```diff
@@ -308,9 +308,12 @@ gRPC services use [HTTP/2 headers](https://github.com/grpc/grpc/blob/master/doc/
 headers:
   :authority: subdomain.host.com
 ```
-
 ## Note
 
 Some [Kubernetes ingress controllers](https://kubernetes.io/docs/concepts/services-networking/ingress/) do not support HTTP/2 fully. As a result, if you are running Ambassador with an ingress controller in front, you may find that gRPC requests fail even with correct Ambassador configuration.
 
 A simple way around this is to use Ambassador with a `LoadBalancer` service, rather than an Ingress controller.
+
+## gRPC-Web
+
+gRPC must be configured to work with [gRPC Web](../../reference/core/ambassador#grpc-web-enable_grpc_web) if you want the same benefits in a browser.
```

Comment:
I am kind of confused about what you are trying to say here since gRPC is not something to configure but a protocol you are using. 

Are you trying say something like "Ambassador also supports the [gRPC-Web](../../reference/core/ambassador#grpc-web-enable_grpc_web) protocol for browser-based gRPC applications."?

Reply:
Yep. Will change the content and re-request.